packages/formation-react/stories/form-controls.stories.mdx

```diff
@@ -1,6 +1,6 @@
 import { Canvas, Meta } from '@storybook/addon-docs/blocks';
 
-<Meta title="Library/Form controls/Usage" />
+<Meta title="Components/Form controls/Usage" />
```

Comment:
This one as well might belong in `Designer Documentation`.

Reply:
Oh, I did miss this one. Good call. :+1: